src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -29,28 +29,29 @@ public Car create(Car car) {
             ResultSet resultSet = insertStatement.getGeneratedKeys();
             if (resultSet.next()) {
                 car.setId(resultSet.getObject(1, Long.class));
-                saveDrivers(car);
```

Comment:
create some boolean helper variable and set it to true in the if body if the resultset wasn't empty